Example/Controllers/ContactsListSettingsTableViewController.swift

```diff
@@ -73,29 +124,53 @@ class ContactsListSettingsTableViewController: UITableViewController {
         }
     }
 
-    func getInputFromAlertWithTextField(title: String? = nil, message: String? = nil, keyboardType: UIKeyboardType? = .asciiCapable, handler: ((UIAlertAction, UITextField?) -> Void)? = nil) {
+    func getInputFromAlertWithTextField(title: String? = nil, message: String? = nil, keyboardType: UIKeyboardType? = .asciiCapable, textFieldDefaultValue: String?, handler: ((UIAlertAction, UITextField?) -> Void)? = nil) {
         DispatchQueue.main.asyncAfter(deadline: .now()) {
             let alert = UIAlertController(title: title,
                 message: message,
                 preferredStyle: .alert)
-            alert.addTextField { (textField) in
-                textField.delegate = self
```

Comment:
there was a bug when using delegate to check if textfield was empty. I replaced the mechanism by a notification listener